src/components/Comments.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getCommentsPerArticle } from "../../utils/api";
+import Collapsible from "./Collapsible";
+
+function Comments() {
+    const [isLoading, setIsLoading] = useState(true);
+    const [comments, setComments] = useState(null);
+    const { item_id } = useParams();
```

Comment:
item_id? Looks like article_id would be a bit more accurate of a name to give this parameter :)